<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewDataSourceDialog</class>
 <widget class="QDialog" name="NewDataSourceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>340</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_dbServerPort">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_dbUser">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_DataSrcName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>31</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>数据源名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_dbServerName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>98</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>300</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>28</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pbt_Confirm">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbt_Cancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>109</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器名称：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_dbPassword">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>71</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>数据库类型：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_dbServerIP">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>IP地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>端口：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_dbType">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>MYSQL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SQLSERVER</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ORACLE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>KINGBASE</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
